### PR TITLE
deploy flowbroker and fix report_manager

### PR DESCRIPTION
**Pull request by danielapetito**
* **Please check if the PR fulfills these requirements**
  * [ ] Tests for the changes have been added (for bug fixes / features)
  * [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)

* **Other information**:

### CODE REVIEW

**Review by gabrielifg — DISMISSED** [review]
Bom trabalho!!!

File: deploy.yaml [hunk 1]
```diff
@@ -351,6 +351,22 @@
         - basic-http
         - report-manager
 
+    - role: rabbitmq
+      tags:
+        - gui-nx
+        - 100k
+        - basic-mqtt
+        - basic-http
+        - flowbroker
```
Comment (marianoleonardo) [inline deploy.yaml:360]: Acho que aqui poderia substituir por rabbitmq

File: deploy.yaml [hunk 2]
```diff
@@ -351,6 +351,22 @@
         - basic-http
         - report-manager
 
+    - role: rabbitmq
+      tags:
+        - gui-nx
+        - 100k
```
Comment (marianoleonardo) [inline deploy.yaml:357]: Essas duas tags devem ser removidas

File: deploy.yaml [hunk 3]
```diff
@@ -351,6 +351,22 @@
...
+        - basic-mqtt
+        - basic-http
+        - flowbroker
+
+    - role: flowbroker
+      tags:
+        - gui-nx
+        - 100k
```
Comment (marianoleonardo) [inline deploy.yaml:365]: Essas duas tags devem ser removidas

**Review by marianoleonardo — CHANGES_REQUESTED** [review]
Deixei alguns comentários

File: deploy.yaml (hunk 2 above)
Comment (danielapetito) [reply deploy.yaml:357]: ok.. removido

File: deploy.yaml (hunk 3 above)
Comment (danielapetito) [reply deploy.yaml:365]: ok.. removido.

File: deploy.yaml (hunk 1 above)
Comment (danielapetito) [reply deploy.yaml:360]: ok.. adicionado.

**Review by marianoleonardo — APPROVED** [review]

**Pull request merged**